proto/eth/v1/beacon_chain.proto

```diff
@@ -293,11 +293,15 @@ message DepositContract {
     string address = 2;
 }
 
+// DEPRECATED: GetWeakSubjectivity endpoint will no longer be supported
 message WeakSubjectivityResponse {
+    option deprecated = true;
 	WeakSubjectivityData data = 1;
 }
 
+// DEPRECATED: GetWeakSubjectivity endpoint will no longer be supported
```

Comment:
```suggestion
// DEPRECATED: GetWeakSubjectivity endpoint is no longer be supported
message WeakSubjectivityResponse {
    option deprecated = true;
	WeakSubjectivityData data = 1;
}

// DEPRECATED: GetWeakSubjectivity endpoint is no longer be supported
```
Here too, i think it's OK to say this endpoint is no longer supported. You could even enhance these messages to say "$ENDPOINT is no longer supported and will be removed in a future release"